File: 2.0/python/numbers/updateNumberLease.py
from callfire.client import CallfireClient

client = CallfireClient('api-login', 'api-password')
client.numbers.updateNumberLease(
    number='19206596476',
    body={
        'callFeatureStatus': 'DISABLED',
        'autoRenew': 'false'
    }
).result()

File: 2.0/php/numbers/updateNumberLease.php
<?php

class ApiClientSample {

    public static function main() {
        $client = \CallFire\Api\DocumentedClient::createClient("login", "password");
        $request = $client->updateNumberLease();
        $request->getOperationConfig()->setPathParameters(array("number" => "19206596476"));
        $body = '{
                    "callFeatureStatus":"DISABLED",
                    "autoRenew":"false"
                 }';
        $request->getOperationConfig()->setBodyParameter($body);
        $result = $client->request($request);
        $json = json_decode($result->getBody());
    }
}

ApiClientSample::main();
